cmake/googletest.cmake

```diff
@@ -13,4 +13,8 @@ FetchContent_Declare(googletest
 )
 # For Windows: Prevent overriding the parent project's compiler/linker settings
 set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
+
+# Disable installation of googletest
```

Comment:
We could guard this with some kind of config option, that maybe is tied into `LD_BUILD_UNIT_TESTS` or something. For now, don't attempt to install googletest when the SDK is installed.